File: Java/algoritmo006/Main.java
package algoritmo006;

import java.util.Scanner;

public class Main {
    public static void executar(){
        Scanner scanner = new Scanner(System.in);

        System.out.println("Pegar faca, bacia e batatas");
        System.out.println("Colocar água na bacia.");

        System.out.print("Quantas batatas você tem? ");
        int qtdBatatas = Integer.parseInt(scanner.nextLine());

        int todasBatatas = qtdBatatas;
        int batataDescascando = 1;

        while (qtdBatatas > 0) {
            System.out.println(
                "Descascando a " + batataDescascando + "ª batata."
            );

            qtdBatatas--;
            batataDescascando++;
        }

        batataDescascando--;

        System.out.println(
            "Tadas as " + todasBatatas + " foram descascadas"
        );
    }

    public static void main(String[] args) {
        executar();
    }
}

File: Python/algoritmo006/main.py
def executar():
    print("Pegar faca, bacia e batatas.")
    print("Colocar água na bacia.")

    qtd_batatas = int(input("Quantas batatas você tem? "))

    todas_batatas = qtd_batatas
    batata_descascando = 1

    while qtd_batatas > 0:
        print(f"Descascando a {batata_descascando}ª batata.")

        qtd_batatas -= 1
        batata_descascando += 1

    batata_descascando -= 1

    print(f"Todas as {todas_batatas} foram descascadas.")

executar()